tests/ImageSharp.Tests/IO/ChunkedMemoryStreamTests.cs

```diff
@@ -30,7 +30,7 @@ public class ChunkedMemoryStreamTests
     [Fact]
```

Comment:
Is it possible to extend these tests to stress the corner(?) cases which were buggy in the previous implementation.

Reply:
Yeah. Tests have been massivly exapanded. We now test reading larger buffers and test encoding to webp for all test images.